waffle_hub/utils/draw.py

```diff
@@ -96,6 +101,31 @@ def draw_instance_segmentation(
     return image
 
 
+def draw_text_recognition(
+    image: np.ndarray,
+    annotation: Annotation,
+    loc_x: int = 0,
+    loc_y: int = 10,
+):
+    if not Path(FONT_NAME).exists():
+        get_file_from_url(FONT_URL, FONT_NAME, True)
```

Comment:
I think the default FONT should be set.

The internet disconnection will cause an error.

I think we can add font file to MANIFEST file (if its file size is small...).